bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/InboxResource.java

```diff
@@ -108,6 +108,13 @@ public Response ignore(@PathParam("thingUID") String thingUID) {
         inbox.setFlag(new ThingUID(thingUID), DiscoveryResultFlag.IGNORED);
         return Response.ok().build();
     }
+    
+    @POST
+    @Path("/unignore/{thingUID}")
```

Comment:
Kai, in that case we can close this PR and I will create new one. I will check all endpoints to have the UID first and the action last.